examples/platforms/nrf528xx/nrf52811/openthread-core-nrf52811-config.h

```diff
@@ -226,6 +236,28 @@
 #define RADIO_CONFIG_SRC_MATCH_EXT_ENTRY_NUM 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_CSL_RECEIVE_TIME_AHEAD
+ *
+ * Reception scheduling and ramp up time needed for the CSL receiver to be ready, in units of microseconds.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_CSL_RECEIVE_TIME_AHEAD
+#define OPENTHREAD_CONFIG_CSL_RECEIVE_TIME_AHEAD 2000
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON
+ *
+ * The total duration, in units of microseconds, required for the CSL receiver to fully receive and acknowledge a frame
```

Comment:
I think this is something more like: "The minimum CSL receive window (in microseconds) required to receive an IEEE 802.15.4 frame."

Total duration indicates the total time it takes to receive and transmit the ack, which should be much longer than the 12 * 16 = 196us value used below.